tests/RobotFramework/tests/pkcs11/private_key_storage.robot

```diff
@@ -233,14 +233,14 @@ Set tedge-p11-server Uri
     Restart Service    tedge-p11-server
 
 Tedge Reconnect Should Succeed
-    ${stdout}=    Execute Command    tedge reconnect c8y
-    RETURN    ${stdout}
+    ${stderr}=    Execute Command    tedge reconnect c8y    stdout=false    stderr=true
+    RETURN    ${stderr}
 
 Tedge Reconnect Should Fail With
     [Arguments]    ${error}
     ${stdout}    ${stderr}=    Execute Command    tedge reconnect c8y    exp_exit_code=!0    stdout=true    stderr=true
     Should Contain    ${stderr}    ${error}
-    RETURN    ${stdout}
+    RETURN    ${stderr}
```

Comment:
RF linting might complain about an unused variable here (`${stdout}`)...since it isn't being used anyware, we don't need to set `stdout=true`.